### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Akash-nath29** [SparklinStar/Numpy-Practice#2] [General]
Hey there, it's [Akash Nath](https://akashnath.netlify.app) this side. You can check my profile from [Here](https://github.com/Akash-nath29) . Have a great day !!

**Comment by SparklinStar** [comment]
Hey Akash! Start by forking this repo and then after creating a folder with your own name make a PR. Under that Folder you will do all the tasks. You can see my folder with the name "Souradip-Pal" for reference.
